selfdrive/ui/qt/api.cc

```diff
@@ -78,17 +81,26 @@ QString CommaApi::create_jwt() {
 RequestRepeater::RequestRepeater(QWidget* parent, QString requestURL, int period_seconds, QVector<QPair<QString, QJsonValue>> payloads, bool disableWithScreen)
   : disableWithScreen(disableWithScreen), QObject(parent)  {
   networkAccessManager = new QNetworkAccessManager(this);
+  connect(networkAccessManager, SIGNAL(sslErrors(QNetworkReply*, const QList<QSslError>&)), this, SLOT(printSslErrors(QNetworkReply*, const QList<QSslError>&)));
+
+  qDebug() << "Device supports OpenSSL: " << QSslSocket::supportsSsl();
 
   QTimer* timer = new QTimer(this);
   QObject::connect(timer, &QTimer::timeout, [=](){sendRequest(requestURL, payloads);});
   timer->start(period_seconds * 1000);
 
   networkTimer = new QTimer(this);
   networkTimer->setSingleShot(true);
-  networkTimer->setInterval(20000); // 20s before aborting
+  networkTimer->setInterval(5000); // 20s before aborting
```

Comment:
Probably want to keep the comment correct or remove it?

Reply:
just testing stuff here :)